app/web/features/communities/events/EventPage.tsx

```diff
@@ -75,6 +77,9 @@ export const useEventPageStyles = makeStyles<Theme, { eventImageSrc: string }>(
       width: "3.125rem",
       height: "3.125rem",
     },
+    snackBar: {
+      fontSize: "0.9rem !important",
```

Comment:
Surely we don't have fontSize CSS rule that strong so that we need to resort to important hacks?